pootle/apps/pootle_misc/checks.py

```diff
@@ -266,56 +267,82 @@ class SkipCheck(Exception):
     pass
 
 
-class ENChecker(checks.TranslationChecker):
+class ENChecker(checks.UnitChecker):
+
+    def __init__(self, checkerconfig=None, excludefilters=None,
```

Comment:
Is it actually necessary to put the `__init__`? I don't see any change in reference to its parent.